lib/elixir/src/elixir.erl

```diff
@@ -67,13 +67,12 @@ start(_Type, _Args) ->
       ok
   end,
 
-  URIs = [{<<"ftp">>, 21},
-          {<<"sftp">>, 22},
-          {<<"tftp">>, 69},
-          {<<"http">>, 80},
-          {<<"https">>, 443},
-          {<<"ldap">>, 389}],
-  URIConfig = [{{uri, Scheme}, Port} || {Scheme, Port} <- URIs],
+  URIConfig = [{{uri, <<"ftp">>}, 21},
+          {{uri, <<"sftp">>}, 22},
```

Comment:
That's very strange, because I did line them up on my machine. Something possibly got reset or I just messed up :) I agree though and will make the change.